src/scss/hpe/_hpe.defaults.scss

```diff
@@ -46,7 +47,7 @@ $brand-graph-colors: (#FF8D6D, #877B75, #2AD2C9, #614767, #617D78);
 
 html,
 .brand-font {
```

Comment:
I think the .brand-font selector should be added to _base.font.scss and we can remove this set of selectors from this file.